### PR TITLE
Create VPC

### PR DESCRIPTION
**Pull request by maripfonde**

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+
+resource "aws_route_table_association" "private0" {
+  subnet_id      = aws_subnet.private0.id
+  route_table_id = aws_route_table.private0.id
+}
+
+resource "aws_route_table_association" "private1" {
+  subnet_id      = aws_subnet.private1.id
+  route_table_id = aws_route_table.private1.id
+}
```
Comment (geoffrey1330) [inline main.tf]: Add newline at the end of file

File: provider.tf [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+      source  = "hashicorp/aws"
+      version = "~> 4.0"
+    }
+  }
+}
+
+# Configure the AWS Provider
+provider "aws" {
+  region = "us-east-1"
+}
```
Comment (geoffrey1330) [inline provider.tf]: Add newline at the end of file

**Review by geoffrey1330 — CHANGES_REQUESTED** [review]

File: main.tf (hunk 1 above)
Comment (maripfonde) [reply main.tf]: Done, thank you.

Comment (geoffrey1330) [reply main.tf]: Still no newline added on the main.tf file.
the provider.tf file have been resolved

Comment (maripfonde) [reply main.tf]: Done, thanks once again.

**Review by geoffrey1330 — APPROVED** [review]

**Pull request merged**